orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/MonitorFront50Task.java

```diff
@@ -138,5 +175,8 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     @JsonProperty("pipeline.name")
     public String pipelineName;
+
+    @JsonProperty("deliveryConfig")
```

Comment:
I don't think this annotation is necessary, since the property name is the same.